android/gradle.properties

```diff
@@ -42,7 +42,7 @@ KEYSTORE_KEY_PASSWORD=password
 # By default we build a mostly universal APK
 ANDROID_ABI_SPLIT=false
 # Some platforms are excluded though
-ANDROID_ABI_INCLUDE=armeabi-v7a;arm64-v8a;x86
```

Comment:
This can be overriten by Make/Jenkins & Nix in some placces.

Reply:
That's fine. It's still a sensible default necessary to be able to use Gradle without all the baggage.

Reply:
Although I do wonder why this architecture is being removed.

Reply:
Initially I removed it to get builds to work because of this : https://github.com/software-mansion/react-native-reanimated/issues/4715#issuecomment-1636855035
Will validate if its needed anymore or not during PR cleanup!